panels/UserPanel.php

```diff
@@ -116,4 +124,19 @@ protected function dataToString($data)
 
         return VarDumper::export($data);
     }
+
+    /**
+     * Returns the array/object that should be set on [[\yii\widgets\DetailView::model]]
+     *
+     * @param IdentityInterface $identity
+     * @return array|object
+     */
```

Comment:
Considering it will get serialized, probably safer to always have `$identityData` be an array. Agree?